src/Nordea/Components/Label.elm

```diff
@@ -216,14 +216,14 @@ stateStyles { hasError } =
 
         outlineStyle =
             Css.batch
-                [ Css.property "box-shadow" ("0rem 0rem 0rem 0.0625rem " ++ color) |> Css.important
+                [ Css.property "box-shadow" ("0rem 0rem 0rem 0.0625rem " ++ color)
                 , outline none
                 ]
 
         styles =
             descendants
-                [ typeSelector "input" [ outlineStyle, borderColor Css.transparent |> Css.important ]
-                , typeSelector "select" [ outlineStyle, borderColor Css.transparent |> Css.important ]
```

Comment:
I'll have a quick look through PH to see if removing this breaks anything. May be good to quickly check CP and esign too?